vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,150 @@
+
+1. Using your browser, examine and save the SSL certificate for your environment. For example, in Chrome, go to any OpenShift console page, click on `View Site Information` next to the URL (it may be labeled `Not secure` ) then go to `Details -> Copy to File` and save the certificate as a Base-64 encoded X.509 file, e.g. `openshift.cer`.
+2. Go to `Workloads → Secrets → Create` and select `Key/value secret`
+3. Set the `Secret Name`, e.g. `entando-ca-cert-secret` 
+4. Set the `Key`, e.g. `openshift.cer`
+5. Set the `Value` by clicking `Browse...` and loading the file you saved in Step 1
+6. Click `Create`
+7. Next go to `Workloads → ConfigMaps` and create or update the ConfigMap named `entando-operator-config`. This is the ConfigMap used by the Operator to configure the deployments. You can [download the Entando Operator template](../devops/entando-operator.md#add-a-new-configmap) as a starting point. Set the "data/entando.ca.secret.name" to match the name from step 3.
+```yaml
+data:
+ entando.ca.secret.name: entando-ca-cert-secret
+ entando.use.auto.cert.generation: 'true'
+```
+8. Click `Create`
+
+Now let's create a new application, just like in Scenario 2 but with OpenShift SSL in place.  
+
+9. Go to `EntandoApp` and click `Create instance`
+   - Keep the default `my-app` for your application name or select your own
+   - Select the EntandoApp version: `7.0`
+   - Set the `Ingress Host Name` as in Scenario 1 above
+   - Keep the default `DBMS` as `postgresql`
+10. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources.
+
+Once the deployment is complete you can confirm that all routes use https with OpenShift's CA. You will likely still see security warnings in the browser.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 4 - PostgreSQL plus self-signed SSL
+This scenario is similar to Scenario 3 but here you'll use a self-signed certificate rather than using OpenShift's Certificate Authority. As a starting point, you can either remove the EntandoApp and Provided Capabilities from the previous scenarios or prepare a new project per steps 1-5 in Scenario 1.
+
+We'll start by creating a self-signed certificate and then prepare the Secrets and ConfigMap to match. There are various ways to create an X.509 self-signed certificate so you can use your preferred mechanism.
+
+1. Using [OpenSSL](https://www.openssl.org/) create a certificate for your application. You'll need to adjust the CN value to match the Ingress Host Name for your project.
+```shell
+openssl req -nodes -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -subj "/CN=entando.apps-crc.testing"
+```
+You should see output similar to this:
+```shell
+Generating a RSA private key
```

Comment:
should "a RSA" be "an RSA"?

Reply:
Yeah, except that's the output from open-ssl so it's a quote.